File: validTri.c
#include <stdio.h>

int main(void){
	int T;

	while(1){
		printf("Input T: ");
		scanf("%d",&T);

		if(T<=0 || T>1000){
			printf("T must be 1 <= T <= 1000\n");
		}else{
			break;
		}
	}

	while(T>0){
		int A;
		int B;
		int C;

		while(1){
			printf("Input A: ");
			scanf("%d",&A);

			if(A<=0 || A>180){
				printf("A must be 1 <= A <= 180\n");
			}else{
				break;
			}
		}

		while(1){
			printf("Input B: ");
			scanf("%d",&B);

			if(B<=0 || B>180){
				printf("B must be 1 <= B <= 180\n");
			}else{
				break;
			}
		}

		while(1){
			printf("Input C: ");
			scanf("%d",&C);

			if(C<=0 || C>180){
				printf("C must be 1 <= C <= 180\n");
			}else{
				break;
			}
		}

		if(A+B+C==180){
			printf("YES\n");
		}else{
			printf("NO\n");
		}

		T--;
	}

	return 0;
}

File: Beginner/validTri/validTri.java
import java.util.Scanner;

public class validTri{
	public static void main(String[] args) {
		int T;
		Scanner scan = new Scanner(System.in);

		while(true){
			System.out.print("Input T: ");
			T = scan.nextInt();

			if(T<=0 || T>1000){
				System.out.println("T must be 1 <= T <= 180");
			}else{
				break;
			}
		}

		while(T>0){
			int A;
			int B;
			int C;

			while(true){
				System.out.print("Input A: ");
				A = scan.nextInt();

				if(A<=0 || A>180){
					System.out.println("A must be 1 <= A <= 180");
				}else{
					break;
				}
			}

			while(true){
				System.out.print("Input B: ");
				B = scan.nextInt();

				if(B<=0 || B>180){
					System.out.println("B must be 1 <= B <= 180");
				}else{
					break;
				}
			}

			while(true){
				System.out.print("Input C: ");
				C = scan.nextInt();

				if(C<=0 || C>180){
					System.out.println("C must be 1 <= C <= 180");
				}else{
					break;
				}
			}

			if(A+B+C==180){
				System.out.println("YES");
			}else{
				System.out.println("NO");
			}

			T--;
		}
	}
}